LayoutTests/editing/selection/delete-selection-with-disconnected-extent.html

```diff
@@ -5,7 +5,10 @@
         testRunner.dumpAsText();
 
     document.querySelector('input').setRangeText('aa', 0, 1, 'end');
-    getSelection().extend(document.createElement('select'));
+    try {
+        getSelection().extend(document.createElement('select'));
```

Comment:
I don’t understand. If this will throw an exception, why call it? Does it have side effects when it throws the exception?

Reply:
I guess an alternative is to add LiveRangeSelectionEnabled=false so that this test will continue to test the crash it was meant to test.

Reply:
Added `<!-- webkit-test-runner [ LiveRangeSelectionEnabled=false ] -->` to this test instead.